configure.py

```diff
@@ -864,7 +1164,12 @@ def get_env(self):
         env = {}
         env["PATH"] = sep.join((self.venv_bin_path, "$(PATH)"))
         env["PKG_CONFIG_PATH"] = self.pkg_config_path
-        if self.host == "Windows":
+        if self.host == "Android":
+            env["PKG_CONFIG_LIBDIR"] = self.pkg_config_path
+            env["PKG_CONFIG_ALLOW_SYSTEM_LIBS"] = "1"
+            env["PKG_CONFIG_ALLOW_SYSTEM_CFLAGS"] = "1"
```

Comment:
mmh wait do you really want to `PKG_CONFIG_ALLOW_SYSTEM_*` for the Android build?

Reply:
We don't. `pkg-config` is either on the build machine system or the build machine venv but will never be installed the host prefix.
Updated.